kong/plugins/aws-lambda/aws-serializer.lua

```diff
@@ -81,6 +86,39 @@ return function(ctx, config)
   local uri = var.upstream_uri or var.request_uri
   local path = uri:match("^([^%?]+)")  -- strip any query args
 
+  local requestContext
+  do
+    local http_version = ngx_get_http_version()
+    local protocol = http_version and 'HTTP/'..http_version or nil
+    local httpMethod = var.request_method
+    local domainName = var.host
+    local domainPrefix = split(domainName, ".")[1]
+    local identity = {
+      sourceIp = var.remote_addr,
```

Comment:
is this IP expected? considering trusted IP's etc.

Reply:
Good point, I double-checked the AWS doc and it says:

> $context.identity.sourceIp | -- |  The source IP address of the immediate TCP connection making the request to API Gateway endpoint.

Maybe we should keep the same behaviour as what AWS does, so the real ip module should not be taken into consideration, it should be `sourceIp = var.realip_remote_addr or var.remote_addr`